css/style.css

```
@@ -0,0 +1,903 @@
+  *,
+*::after,
+*::before {
+    box-sizing: border-box;
+}
+
+html {
+    scroll-behavior: smooth;
+    /* свойство scroll-behavior не наследуется, применяется к прокручиваемым блокам */
+}
+
+ul {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+    color: inherit;
+    display: inline-block;
+}
+
+.header {
+    font-family: Montserrat, sans-serif;
+    height: 806px;
+    background-color: #eff0f2;
+}
+
+.container {
+    width: 1140px;
+    margin: 0 auto;
+}
+
+.header_logo {

```

Incorrect class naming here and beyond. It isn't meet BEM methodology. Fix it all.

---

Thanks!
i fixed all
```suggestion 
  .header_logo {
  .header-logo {
  
  .header_text__style {
  .header-text__style {
  
  and other...
```